lib/modules/dosomething/dosomething_reportback/includes/dosomething_reportback_file.inc

```diff
@@ -58,17 +58,16 @@ class ReportbackFileEntity extends Entity {
       // Store that instead.
       $this->review_source = $values['source'];
     }
-    if ($this->status === 'flagged') {
```

Comment:
Yeah, so the `flag` function is now called on every reportback file review, and sets the value to `0` or `1` 
since the default is now null